### ISSUE TITLE
Stack overflow for a limit evaluation

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#1698]
Imported from SourceForge on 2024-07-04 23:19:20
Created by **[apunto13](https://sourceforge.net/u/apunto13/)** on 2019-10-09 15:28:54
Original: https://sourceforge.net/p/maxima/bugs/3589

---

By evaluating:

`limit((sqrt(x)-2)*log(1-sqrt(x)/2),x,4,minus);`

we get 

~~~
Maxima encountered a Lisp error:

 Condition in MACSYMA-TOP-LEVEL [or a callee]: INTERNAL-SIMPLE-ERROR: Bind stack overflow.
~~~

Maxima version: "5.42.1"
Maxima build date: "2019-01-15 11:28:41"
Host type: "x86_64-pc-linux-gnu"
Lisp implementation type: "GNU Common Lisp (GCL)"
Lisp implementation version: "GCL 2.6.12"

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-04 23:19:21
Created by **[apunto13](https://sourceforge.net/u/apunto13/)** on 2019-10-09 15:44:41
Original: https://sourceforge.net/p/maxima/bugs/3589/#e22b

---

Solved by patch proposal [#90](https://sourceforge.net/p/maxima/patches/90/)

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-04 23:19:25
Created by **[dgildea](https://sourceforge.net/u/dgildea/)** on 2019-10-19 14:12:57
Original: https://sourceforge.net/p/maxima/bugs/3589/#014d

---

- **status**: open --> closed
- **assigned_to**: Dan Gildea

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-04 23:19:28
Created by **[dgildea](https://sourceforge.net/u/dgildea/)** on 2019-10-19 14:12:58
Original: https://sourceforge.net/p/maxima/bugs/3589/#39aa

---

fixed by [78eedaa0993d63670e83fcb89901c36d71f21f46]
